server/app.js

```diff
@@ -219,58 +189,71 @@ app.get('/user/:username', async (req, res, next)=>{
 				console.error(err);
 			});
 
-		if(googleBrews) {
+		if(googleBrews && googleBrews.length > 0) {
+			for (const brew of brews.filter((brew)=>brew.googleId)) {

```

Comment:
Need to include `googleId` in the `fields` array above. Otherwise we are not retrieving `googleId` for any brews, and we can't compare stubs to google brews. Hence we end up with both the stub and the google brew appearing in the user page.

Reply:
Ahh I missed that projection, my bad

Reply:
Should be fixed now!